index.html

```diff
@@ -44,44 +91,139 @@
                         <div class="hp"></div>
                         <div class="hp"></div>
                     </div>
-                    <div class="card"></div>
+                    <div>
+                        <div class="portait-wrap">
+                            <div class="portait"></div>
+                        </div>
+                        <div class="buff-list right">
+                            <div class="buff-wrap">
+                                <div class="buff buff-1"></div>
+                            </div>
+                            <div class="buff-wrap">
+                                <div class="buff buff-2"></div>
+                            </div>
+                            <div class="buff-wrap">
+                                <div class="buff buff-3"></div>
+                            </div>
+                            <div class="buff-wrap">
+                                <div class="buff buff-4"></div>
+                            </div>
+                            <div class="buff-wrap">
+                                <div class="buff buff-5"></div>
+                            </div>
+                        </div>
+                        <div class="weapon-wrap">
+                            <div class="weapon"></div>
+                        </div>
+                    </div>
                 </div>
                 <div class="player">
-                    <div class="hp-bar">
+                    <div class="hp-bar hp-bar-right">
                         <div class="hp"></div>
                         <div class="hp"></div>
                         <div class="hp"></div>
                         <div class="hp"></div>
                         <div class="hp"></div>
                     </div>
-                    <div class="card"></div>
+                    <div>
+                        <div class="portait-wrap">
+                            <div class="portait portait-right"></div>
+                        </div>
+                                                <div class="buff-list">
+                            <div class="buff-wrap">
+                                <div class="buff buff-reverse-1"></div>
```

Comment:
can you avoid hard numeration in classes? looks like we have only five buffs max, no more